examples/mip/bilinear.py

```diff
@@ -0,0 +1,28 @@
+# Inspired by https://www.gurobi.com/documentation/9.5/examples/bilinear_py.html#subsubsection:bilinear.py
+#
+# This example formulates and solves the following simple bilinear model:
+#  maximize    x
+#  subject to  x + y + z <= 10
+#              x * y <= 2         (bilinear inequality)
+#              x * z + y * z = 1  (bilinear equality)
+#              x, y, z non-negative (x integral in second version)
```

Comment:
For my understanding, this is not a MIP, but just an almost-linear problem? In other words, there are no integer constraints. In other other words, you could solve this with the existing minimizers

Reply:
Second note, I don't see the non-negative boundaries in the code below